src/BenchmarksApps/TodosApi/appsettings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Authentication": {
+    "Schemes": {
+      "Bearer": {
+        "ValidAudiences": [
+          "http://localhost:5054"
+        ],
+        "ValidIssuer": "dotnet-user-jwts"
```

Comment:
(nit) do we want `dotnet-user-jwts` in the production appsettings.json?  or just in development.json?

Reply:
It's mostly for convenience for this test/benchmark app. If we don't use `dotnet-user-jwts` we need to create another thing that produces JWTs with a different issuer name.